src/index.html

```diff
@@ -1,13 +1,283 @@
+              </article>
+
+              <article class="category">
+                <div class="category__photos">
+                  <a href="#" class="category__link category__link--wide">
+                    <img
+                    class="category__photo"
+                    src="images/categories/Accessories - wide.png"
+                    alt="Accessories"
+                    >
+                  </a>
+
+                  <a href="#" class="category__link category__link--square">
+                    <img
+                    class="category__photo"
+                    src="images/categories/Accessories - square.png"
+                    alt="Accessories"
+                    >
+                  </a>
+                </div>
+                <h3>Accessories</h3>
+              </article>
+              <a
+                class="button button__view"
+                href="#"
+                data-qa="hover"
+               >
+              View all
+              </a>
+            </div>
+          </section>
+
+          <section>
+            <div class="center-background">
+              <img
+                class="Image__fon"
+                src="./images/categories/Image fon.png"
+                alt="Image fon"
+              >
+            </div>
+          </section>
+
+          <section class="how-to-buy" id="how-to-buy">
+            <h2 class="
+            how-to-buy__title
+            section-title
+            section-title--align--left"
+            >
+              About us
+            </h2>
+            <div class="how-to-buy__content">
+              <div class="how-to-buy__description">
+                <p> We`re a London-based tech company on a mission to remove barriers between people and technology.
+                </p>
+                <p>
+                  To do this, we`re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+                  <br>
+                  No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products we`re proud to share with our friends and family. Simple.
+                </p>
+                <p>
+                  Because beautiful tech shouldn`t be complicated.
+                </p>
+              </div>
+            </div>
+          </section>
+
+          <section class="contacts" id="contacts">
+            <h2 class="section-title">
+              Contact us
+            </h2>
+
+            <div class="contact">
+              <div class="contact__titles">
+                <form action="action_page.php">
+
+                  <input type="text" id="fname"
+                    name="firstname"
+                    placeholder="Name">
+
+                  <input type="text" id="lname"
+                    name="lastname" placeholder="Email">
+
+                  <textarea id="subject" name="subject"
+                    placeholder="Message">
+                  </textarea>
```

Comment:
```suggestion
                  <textarea
                     id="subject"
                     name="subject"
                     placeholder="Message"
                   ></textarea>
```
close bracket for open tag and closing tag should be on one line. it will prevent additional spaces in textarea